### PR TITLE
Sonoma Web Apps support. Lit 3

### PR DESCRIPTION
**Pull request by khmyznikov**
Major release:
- PWA Install dialog for Sonoma Safari (need to test first)
- Upgrade to Lit v3
- New translation strings AI translated (need to verify)
- New Hide Screenshots property
- iOS start_url hack

### CODE REVIEW

**Comment by khmyznikov** [comment]
Calling everyone who contributed to features and translations. Can you verify your language strings?
And also, if someone have Sonoma device, can you check in Safari? I will do this soon but it's not very handy for me.
@ShoWerZen @dartess @Blankjr @s0xDk @jivanf @MertcanDinler @hoRacy @benfavre @Georgegipa

**Comment by s0xDk** [comment]
`uk` and `ru` looks fine.

will check Safari in Sonoma now...

**Comment by s0xDk** [comment]
Works like a charm in Sonoma/Safari! 🔥

**Comment by jivanf** [comment]
I can't test on Sonoma, but `es` LGTM

Thank you for this release!

File: i18n/tr.xlf [hunk 1]
```diff
@@ -1,47 +1,63 @@
...
+        <source>Add to Home Screen</source>
+        <target>Ana Ekrana Ekle</target>
+      </trans-unit>
+      <trans-unit id="sfea652f6580ff086">
+        <source>This site has app functionality.</source>
+        <target>Bu site uygulama işlevselliğine sahiptir.</target>
+      </trans-unit>
+      <trans-unit id="se740f75b95a51807">
+        <source>Add it to your Dock for extensive experience and easy access.</source>
+        <target>Geniş bir deneyim ve kolay erişim için Dock’a ekleyin.</target>
```
Comment (MertcanDinler) [inline i18n/tr.xlf:43]:
```suggestion
        <target>Daha iyi bir deneyim ve kolay erişim için Dock’a ekleyin.</target>
```

File: i18n/tr.xlf [hunk 2]
```diff
@@ -1,47 +1,63 @@
...
+      </trans-unit>
+      <trans-unit id="s182ab2d6c997515f">
+        <source>Add it to your Home Screen for extensive experience and easy access.</source>
+        <target>Geniş bir deneyim ve kolay erişim için Ana Ekranınıza ekleyin.</target>
```
Comment (MertcanDinler) [inline i18n/tr.xlf:47]:
```suggestion
        <target>Daha iyi bir deneyim ve kolay erişim için Ana Ekranınıza ekleyin.</target>
```

File: i18n/tr.xlf [hunk 3]
```diff
@@ -1,47 +1,63 @@
...
+      </trans-unit>
+      <trans-unit id="sa7551383d1897fcc">
+        <source>2) Press Add to Dock</source>
+        <target>2) Dock’a Ekle’ye basın</target>
```
Comment (MertcanDinler) [inline i18n/tr.xlf:51]:
```suggestion
        <target>2) Dock’a ekleye basın</target>
```

File: i18n/tr.xlf [hunk 4]
```diff
@@ -1,47 +1,63 @@
...
+      </trans-unit>
+      <trans-unit id="s922329d6f6213590">
+        <source>Add to Dock</source>
+        <target>Dock’a Ekle</target>
+      </trans-unit>
+      <trans-unit id="sba52286c21552a4e">
+        <source>Install it on your device for extensive experience and easy access.</source>
+        <target>Cihazınıza yükleyin, böylece geniş bir deneyim ve kolay erişime sahip olursunuz.</target>
```
Comment (MertcanDinler) [inline i18n/tr.xlf:59]:
```suggestion
        <target>Daha iyi bir deneyim ve kolay erişim için cihazınıza yükleyin.</target>
```

**Review by MertcanDinler — CHANGES_REQUESTED** [review]

**Comment by hoRacy** [comment]
PL looks fine. Thank for your work!

**Comment by Blankjr** [comment]
DE string translations looking fine.  Appreciate your efforts!

**Pull request merged**